docs/examples/ParticleFlow.py

```diff
@@ -201,8 +201,9 @@
         [siap_gen], associator=TrackToTruth(association_threshold=np.inf))
     metric_manager.add_data(tracks={track}, groundtruth_paths={truth})
 
-    pa[filter] = {metric for metric in metric_manager.generate_metrics()
-                  if metric.title.startswith("SIAP Position Accuracy at times")}.pop()
+    metrics = metric_manager.generate_metrics()
+    pa[filter] = {metrics.get(metric) for metric in metrics
+                  if metric.startswith("SIAP Position Accuracy at times")}.pop()
```

Comment:
Since you can work with keys now, this would be simpler:
```suggestion
    pa[filter] = metrics.get("SIAP Position Accuracy at times")
```